craco.config.js

```diff
@@ -1,13 +1,20 @@
 module.exports = {
     webpack: {
-        configure:{
-            // See https://github.com/webpack/webpack/issues/6725
-            module:{
+        configure: {
+            module: {
                 rules: [{
                     test: /\.wasm$/,
                     type: 'javascript/auto',
-                }]
-            }
-        }
-    }
+                    use: [{ loader: 'file-loader' }]
+                }],
+            },
+            resolve: {
+                fallback: {
+                    'path': require.resolve('path-browserify'),
+                    'crypto': require.resolve('crypto-browserify'),
+                    'stream': require.resolve('stream-browserify')
```

Comment:
Do we actually use path, crypto and stream anywhere ?

Reply:
that a good question, and honestly I don"t really know how webpack handle wasm, nor how sql.js is built.
My point is everything was working fine for me with Webmact4, and when updating to Webpack5 I had 3 successive errors saying "path", "crypto" and "stream" are not included as polyfill (comming from sql.js/dist), , and adding these lines fixed the error and everything run again smoothly

Reply:
We should probably avoid encouraging users to bundle three dependencies they don't need with their code.

Reply:
In addition to `path`, `crypto` and `stream`, I've just encountered another error related to `fs`. It seems that setting the fallback of `fs` to `false` got it working for me. Is there a chance other dependencies can be set to `false` as well?